sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TokenTest.java

```diff
@@ -0,0 +1,82 @@
+package com.microsoft.appcenter.storage;

```

Comment:
Shouldn't the test file be one level deeper in `.test` package and folder?

Reply:
nope... according to other test case, it's all under the module level